web/packages/teleterm/src/services/tshd/types.ts

```diff
@@ -59,6 +59,7 @@ export interface Server extends apiServer.Server.AsObject {
 
 export interface App extends apiApp.App.AsObject {
   uri: uri.AppUri;
+  addrWithProtocol: string;
```

Comment:
That doesn't seem to be true, `addrWithProtocol` exists on `App` defined in `ClustersService`, but not this one.

Reply:
Sorry, I added it by a mistake.